lib/lita/handlers/reviewme.rb

```diff
@@ -120,7 +121,8 @@ def next_reviewer(room)
       end
 
       def github_comment(reviewer)
-        ":eyes: @#{reviewer}"
+        msg = config.github_comment_template || ":eyes: %s"
```

Comment:
Is there a way to set this configuration to a default value in the library? I'd probably rather have it default to something rather than have to check for the `nil` here.